packages/Button/stories/examples/ButtonSubmit.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ButtonStory } from "../Button.stories.styles";
+import Button from "../../src";
+
+const ExampleSubmitStory = () => {
+  return (
+    <ButtonStory>
+      <form
+        onSubmit={event => {
+          alert("submitted");
+          event.preventDefault();
+        }}
+      >
+        <input type="text" defaultValue="press enter while focus" />
+        <Button isSubmit isSemantic={false}>
```

Comment:
Excuse me, but... what is this?
![image](https://user-images.githubusercontent.com/30345591/71717435-66fa6480-2e18-11ea-900e-0717051b7d36.png)

What the reason to have `isSemantic={false}`?

As I understood, we render Button as `<span>` and then trying to emulate `<button>` behavior on `<span>`. But why?

Reply:
It is for the same reason, style conflicts that are hard to fix from some applications in our ecosystem. In this case, the submit button behaves correctly as a button when it is render as span, but was lacking the ability to fire the onSubmit event, this addresses that. 

I guess eventually someday we will be able to deprecate the isSemantic prop.

Reply:
@DanilAgafonov 
![image](https://user-images.githubusercontent.com/93752/71745212-5816b080-2e1e-11ea-9202-332441914e23.png)